<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,1">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,2,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>模板列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MBcomboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="deleteMB">
           <property name="text">
            <string>删除当前模板</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>底料长度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tableX">
             <property name="text">
              <string>90</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>底料宽度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tableY">
             <property name="text">
              <string>90</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="certen">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>图案大小</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>90</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>80</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>70</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>模板名称</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="modelName"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="saveModel">
           <property name="text">
            <string>保存模板</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="debugLabel">
     <property name="text">
      <string>debug</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
